### ISSUE TITLE
ctrl+alt+oem2 (Spanish Curly Brackets closing) opens commentary editing

### ISSUE THREAD

**Issue by Nerioth11** [kasecato/vscode-intellij-idea-keybindings#117]
Whenever I use the curly brackets closing in the spanish keyboard this opens this comment editing thing:
`<!--  -->`
Is there anyway to solve it? I really love this extension but this is very annoying when coding in Angular. Thanks!

**Comment by kasecato** [comment]
Thank you for using, @Nerioth11 !
How do you bind comment out on Spanish keyboard?
- US ASCII
  - win or linux: `ctrl+alt+/`
  - macos: `cmd+alt+/`

Related issue with: https://github.com/Microsoft/vscode-atom-keybindings/issues/40

**Comment by Nerioth11** [comment]
Hi kasecato! Seems it's Ctrl+Alt+NumPad_Divide in the VS Code Default Shortcuts. Would love to get this fixed!! thanks for your atention ^^

**Comment by kasecato** [comment]
Hi @Nerioth11 ,
I just remembered I had fixed these shortcuts for Nordic Keyboards (See #5). 
> Commenting and uncommenting in Intellij works with numpad and in the best way on Nordic Keyboards.

Hi @mathiasfrohlich , long time no see :)
I'd like to disable `Ctrl+Alt+NumPad_Divide` (Comment/uncomment with block comment). Can we disable it? Please see #5 for more details.

**Comment by Nerioth11** [comment]
So to what file should I add this then?:
```
{
    "key": "ctrl+numpad_divide",
    "mac": "cmd+numpad_divide",
    "command": "editor.action.commentLine",
    "when": "editorTextFocus"
    "intellij": "Comment/uncomment with line comment using Numpad"

}
```

Because if it's already fixed for me it isn't working, at least with the version from the VS Plugin store :/

**Comment by kasecato** [comment]
Closed by #305
